### PR TITLE
ci: more specific ci config folder names

### PR DESCRIPTION
**Pull request by subhamkrai**
Signed-off-by: subhamkrai <srai@redhat.com>

**Description of your changes:**

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure)).
- [ ] **Skip Tests for Docs**: If this is only a documentation change, add the label `skip-ci` on the PR.
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

**Review by travisn — CHANGES_REQUESTED** [review]
- Could you add some comment in the canary-test-config/action.yaml to point out why it is needed separate from the integration-test-config?
- Please add a more specific commit title such as "ci: more specific ci config folder names"

**Comment by subhamkrai** [comment]
> * Could you add some comment in the canary-test-config/action.yaml to point out why it is needed separate from the integration-test-config?
added comment in integration-test-config/action.yaml file
> * Please add a more specific commit title such as "ci: more specific ci config folder names"

added

File: .github/workflows/integration-test-config/action.yaml [hunk 1]
```diff
@@ -28,6 +28,7 @@ runs:
     shell: bash --noprofile --norc -eo pipefail -x {0}
     run: tests/scripts/github-action-helper.sh print_k8s_cluster_status
 
+  # This setp is not required for canary integration tests so keeping this new file
```
Comment (travisn) [inline .github/workflows/integration-test-config/action.yaml:31]: Just to be clear, is this valid?
```suggestion
  # This step is required for all the integration tests except the canary tests
```

**Review by travisn — CHANGES_REQUESTED** [review]

File: .github/workflows/integration-test-config/action.yaml (hunk 1 above)
Comment (subhamkrai) [reply .github/workflows/integration-test-config/action.yaml:31]: yes

**Review by travisn — APPROVED** [review]

**Pull request merged**